src/demo/pages/BlockchainTx.css

```diff
@@ -191,4 +191,10 @@
   right: -11px;
   height: 5px;
   background-color: var(--primary);
+}
+
+@media only screen and (min-width: 375px) and (max-width: 767px) {
+  .MuiSvgIcon-root {
```

Comment:
use of pascal case here

Reply:
it's the package mui class so i need to use it as it is

Reply:
if i added custom class with i need to use !Important